Automatic Validation ended with:
> 2024-03-11T16:41:41.0100749Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\h\Headlamp\Headlamp\0.23.0\0.23.0', expected 'manifests\h\Headlamp\Headlamp\0.23.0') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\h\Headlamp\Headlamp\0.23.0\0.23.0', expected 'manifests\h\Headlamp\Headlamp\0.23.0'

(Automated response - build 869.)

--- a/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.installer.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.installer.yaml
@@ -1,0 +1,23 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.23.0
+Installers:
+  - Architecture: x64
+    InstallerLocale: en-US
+    InstallerType: exe
+    InstallerUrl: https://github.com/headlamp-k8s/headlamp/releases/download/v0.23.0/Headlamp-0.23.0-win-x64.exe
+    InstallerSha256: abd6f11ebf6d4e55c806ebc4c361c51f7d9340548adc2666e98cae828c473c76
+    Scope: user
+    InstallModes:
+      - interactive
+      - silent
+    InstallerSwitches:
+      Silent: /S
+      SilentWithProgress: /S
+    AppsAndFeaturesEntries:
+      - DisplayName: Headlamp
+        DisplayVersion: 0.23.0
+        Publisher: Headlamp
+        InstallerType: exe
+    ReleaseDate: 2024-03-06
+ManifestType: installer
+ManifestVersion: 1.5.0

--- a/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.locale.en-US.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.locale.en-US.yaml
@@ -1,0 +1,27 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.23.0
+PackageLocale: en-US
+Publisher: Headlamp
+PublisherUrl: https://headlamp.dev
+PackageName: Headlamp
+License: Apache 2.0
+LicenseUrl: https://github.com/headlamp-k8s/headlamp/blob/main/LICENSE
+ShortDescription: Kubernetes web UI.
+Description: A Kubernetes web UI that is fully-featured, user-friendly and extensible.
+Tags:
+  - kubernetes
+  - plugins
+  - kinvolk
+  - headlamp
+  - dashboard
+  - ui
+  - web
+  - monitoring
+  - logging
+  - cncf
+Documentations:
+  - DocumentLabel: headlamp documentation
+    DocumentUrl: https://headlamp.dev/docs/latest
+ReleaseNotesUrl: https://github.com/headlamp-k8s/headlamp/releases/tag/v0.23.0
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0

--- a/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.23.0/Headlamp.Headlamp.yaml
@@ -1,0 +1,5 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.23.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0

